lib/elixir/test/elixir/string_io_test.exs

```diff
@@ -140,7 +140,7 @@ defmodule StringIOTest do
   test "IO.binwrite with utf8" do
     pid = start("")
     assert IO.binwrite(pid, "あいう") == :ok
-    assert contents(pid) == {"", "あいう"}
+    assert contents(pid) == {"", <<195, 163, 194, 129, 194, 130, 195, 163, 194, 129, 194, 132, 195, 163, 194, 129, 194, 134>>}
```

Comment:
Are people relying on this bug? The `IO.binwrite` docs say not to use it on a `:unicode` device.